src/app/core/nav-menu/components/license/license.component.ts

```diff
@@ -0,0 +1,34 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+  selector: 'cs-license',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="license">
+      {{ 'NAVIGATION_SIDEBAR.LICENSE.LICENSE_1' | translate: { year: getCurrentYear() } }}
+      <a href="https://bitworks.software/" target="_blank">
+        {{ 'NAVIGATION_SIDEBAR.LICENSE.LICENSE_2' | translate }}
+      </a>
+      <br>
+      {{ 'NAVIGATION_SIDEBAR.LICENSE.LICENSE_3' | translate }}
+    </div>
+  `,
+  styles: [`
+    .license {
+      font-size: 12px;
+      padding: 5px;
+      line-height: 1;
+      text-align: center;
+    }
+
+    a {
+      color: inherit;
+    }
+  `]
+})
+
+export class LicenseComponent {
+  public getCurrentYear(): string {
```

Comment:
Do you guys have some kind of style guide? Why are you using `public` modifier explicitly here, but for properties like `routes: Route[];` you omit them?

Reply:
Style guide in progress #1061 
But we don't have any explicit rule for that. I did not add visibility to routes as this is an input property, also I didn't add visibility to output props.